handbook/business-operations/vendor-questionnaires.md

```diff
@@ -6,19 +6,27 @@
 | Will Fleet allow us to conduct our own penetration test?   | Yes                                                               |
 
 
-## Application security
+## [Application security](https://fleetdm.com/docs/using-fleet/application-security#application-security)
```

Comment:
Instead of wrapping this heading in a link, you can include the link below.  (Also answer: why would I click it?  What is at this link?  What is it for?)
Sam+Mike 2023-07-03

Reply:
The link is a cross-reference to existing documentation on the subject. Instead of duplicating content, the user would go to the originating source.

Reply:
https://github.com/fleetdm/fleet/pull/12366/files#r1253738154